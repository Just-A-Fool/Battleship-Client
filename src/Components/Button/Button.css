Button {
  font-size: 1.5rem;
  margin: 10px;
  color: black;
  background-color: rgb(162, 188, 214, 0.6);
  padding: 0.5rem;
  font-family: 'Roboto Condensed', sans-serif;
  font-weight: bold;
  border-radius: 5px;
}

button a {
  text-decoration: none;
  color: black;
}

button a:hover,
button:hover {
  color: white;
  background-color:rgb(16, 43, 86, 0.6);
}
