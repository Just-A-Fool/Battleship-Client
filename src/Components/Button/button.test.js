import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Button from './Button'

describe(`Button Component`, () => {
    it(`Renders without errors`, () => {
        const div = document.createElement('div');
        ReactDOM.render(<Button /> , div)
        ReactDOM.unmountComponentAtNode(div)
    }),

    it(`Displays a button`, () => {
        const wrapper = shallow(<Button />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })

})