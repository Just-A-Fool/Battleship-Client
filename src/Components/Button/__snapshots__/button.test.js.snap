// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Displays a button 1`] = `
<button
  className="Button"
/>
`;
